openlibrary/macros/databarAuthor.html

```diff
@@ -75,10 +75,10 @@ <h2>$_('Create a new list')</h2>
         <li>
             <span class="image">
             $ cover = "//covers.openlibrary.org/b/olid/%s-M.jpg" % page.cover_edition_key if page.cover_edition_key else "https://openlibrary.org/images/icons/avatar_book-sm.png"
-            <a href="$page.url"><img src="$cover" width="32" alt="Cover of: $page.title$(': ' + page.subtitle if page.get('subtitle', None) else '')" title="Cover of: $page.title$(': ' + page.subtitle if page.get('subtitle', None) else '')"/></a>
+            <a href="$page.url"><img src="$cover" width="32" alt="$_('Cover of: %(title)s', title=f'{page.title}:{page.subtitle} if page.get('subtitle', None) else page.title)" title="$_('Cover of: %(title)s', title=f'{page.title}:{page.subtitle} if page.get('subtitle', None) else page.title)"/></a>
```

Comment:
```suggestion
            <a href="$page.url"><img src="$cover" width="32" alt="$_('Cover of: %(title)s', title=f'{page.title}:{page.subtitle}' if page.get('subtitle', None) else page.title)" title="$_('Cover of: %(title)s', title=f'{page.title}:{page.subtitle}' if page.get('subtitle', None) else page.title)"/></a>
```

Added missing closing quotes, but there's an even better approach!

Instead of repeating the variable declaration here, you could add it, and perhaps also the cover declaration right above (ensuring it doesn't break anything), to the top of the file.

Sample declaration at top of file ([author_photo.html]([here](https://github.com/internetarchive/openlibrary/blob/4821b25e23e421e5b9e2ab4caa31d3f5c855e70b/openlibrary/templates/covers/author_photo.html#L3))):
```
$def with(author, size="M")

$ author_thumbnail_url = author.get_photo_url(size)
$ author_photo_url = author.get_photo_url("L")
```

What you could change the top of `databarAuthor.html` to:
```
$def with (page)

$ cover = "//covers.openlibrary.org/b/olid/%s-M.jpg" % page.cover_edition_key if page.cover_edition_key else "https://openlibrary.org/images/icons/avatar_book-sm.png"
$ title = f'{page.title}:{page.subtitle}' if page.get('subtitle', None) else page.title)
```

And then both the alt and the title could be re-written as:
```
"$_('Cover of: %(title)s', title=title)"
```